.github/workflows/node.js.yml

```diff
@@ -0,0 +1,49 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x, 14.x, 15.x]
+        build-how: ["node", "browser", "browser -n"]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    # normal browser build
+    - run: npm install
+    - if: ${{ contains(matrix.build-how, 'browser') }}
```

Comment:
just FYI, an `if` should be attached to a "step" and not by itself

Reply:
also, `${{ }}` notation is automatically assumed in an `if` so you can just do contains

Reply:
> just FYI, an if should be attached to a "step" and not by itself 

Seems to work with multiple ifs inside `steps` doing different things.